aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -612,4 +612,14 @@ module aptos_framework::aptos_governance {
         initialize(&account, 1, 2, 3);
         update_governance_config(&account, 10, 20, 30);
     }
+
+    #[verify_only]
```

Comment:
Does this strip this function from the deployed bytecode?

Reply:
I believe so. @meng-xu-cs , could you confirm that "verify_only" functions wouldn't be included in the compiled bytecode to deploy?

Reply:
Yes, confirming that a verify_only function will not show up in deployable code